presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java

```diff
@@ -521,6 +521,35 @@ public void testBinaryFunctions()
         // from_base64, to_base64.
         assertQuery("SELECT from_base64(to_base64(cast(comment as varbinary))) FROM orders");
 
+        // from_big_endian_32, to_big_endian_32.
+        assertQuery("SELECT to_big_endian_32(null), from_big_endian_32(to_big_endian_32(null))");
```

Comment:
Write a lambda function for {assertQuery(....)) and invoke with the value parameter to make this code more readable.

Reply:
I have updated the code, thanks for the suggestion.